include/oneapi/dpl/pstl/algorithm_impl.h

```diff
@@ -71,19 +71,44 @@ __pattern_any_of(_ExecutionPolicy&&, _ForwardIterator __first, _ForwardIterator
     return __internal::__brick_any_of(__first, __last, __pred, __is_vector);
 }
 
+template <class _Tag, class _ExecutionPolicy, class _ForwardIterator, class _Pred>
+bool
+__pattern_any_of(_Tag, _ExecutionPolicy&&, _ForwardIterator __first, _ForwardIterator __last, _Pred __pred) noexcept
```

Comment:
It seems that we need to have a serial tag (depending on isVector) here, not general type `_Tag`. (and everywhere in the similar places below)

Reply:
Ok, static_assert is acceptable approach here.
(General type tag was designed for dispatching to "defualt" serial implementation in case of non-oneDPL tag type)